shopelectro/migrations/0037_populate_matrixblock.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-07-04 09:54
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def migrate_forward(apps, schema_editor):
+    MatrixBlock = apps.get_model('shopelectro', 'MatrixBlock')
+    Category = apps.get_model('shopelectro', 'Category')
+
+    # only default managers work in migration, so we can't use .active()
+    for c in Category.objects.filter(page__is_active=True, level=0):
```

Comment:
```suggestion
    for c in Category.objects.filter(level=0):
```

we won't filter not active pages. Views should filter them